.cirrus.yml

```diff
@@ -0,0 +1,53 @@
+container:
+  image: ghcr.io/cirruslabs/android-sdk:33
+  kvm: true
+  cpu: 8
+  memory: 16G
+
+check_android_task:
```

Comment:
Maybe we should give this a more speaking name? `Cirrus CI instrumentation tests`?

Reply:
Done.